swagger.yaml

```diff
@@ -1359,6 +1366,35 @@ definitions:
       size:
         type: string
         description: "The size of the file in bytes"
+  Alert:
+    description: "A single alert, &#x2603;"
```

Comment:
I don't think we need the 3  &#x2603; in the swagger spec

Reply:
lol, ian told me to put it there....I forgot I did that 👍